docs/source/install.rst

```diff
@@ -1,42 +1,145 @@
 Installing Quipucords
 =====================
-Quipucords is delivered in two parts, a command line tool as an RPM package and a server as a container image. The following instructions describe how to install the parts of Quipucords.
+Quipucords is delivered in two parts, a command line tool as an RPM package and a server as a container image. The following instructions describe how to install the parts of Quipucords using an scripted installation or step by step commands.
```

Comment:
parts of Quipucords using an scripted installation --> using a scripted installation

Reply:
done